RE mcmc plots

jessicaguo opened the pull request:
Assuming that the results of set.MA.trt will be available in the pft folder, this function takes 4 inputs (priors, data, posteriors, and treatment matching table), makes each one a list object of the same length (number of traits), then creates a plot comparing each trait by treatment. 

There is a subfunction plot.prior.posterior() that is used inside 'mapply' to create a list of ggplot objects. Each object is printed out and saved as "traitname.jpeg"

Latest set of commits accounts for the new set_MA_trt.R, reorders the facets by site and treatment, and adjusts the x-axis limits from (0, max(0.975 of prior, posteriors, and max(data)).

dlebauer commented on BioCro/DARPA/set.MA.trt.R:
```diff
@@ -0,0 +1,90 @@
+  library(coda)
+  
+  #load jagged.data and trait.mcmc
+  load(paste0(settings$outdir, "pft/SetariaWT_ME034/jagged.data.Rdata"))
```
```suggestion
  # this will only work for a single PFT run e.g. will break w/ ED2 but ok for now
  load(file.path(settings$outdir, "pft", settings$pft$name, "jagged.data.Rdata"))
```

jessicaguo commented:
Updated to better reflect the tidyverse style guide.. Removed the y-axis labels and ticks for a cleaner graphic. 
![quantum_efficiency](https://user-images.githubusercontent.com/12022076/101843356-4302d100-3b07-11eb-96b2-a12983f4b255.jpg)

jessicaguo commented:
New plot examples. 
![SLA](https://user-images.githubusercontent.com/12022076/102915973-f640c380-443f-11eb-8e2a-c12baee37c94.jpg)
![quantum_efficiency (1)](https://user-images.githubusercontent.com/12022076/102915976-f640c380-443f-11eb-902e-49bcd1da85df.jpg)

KristinaRiemer commented on BioCro/DARPA/plot_MA.R:
```diff
@@ -0,0 +1,87 @@
+  load(file.path(settings$pfts$pft$outdir, "trt.match.Rdata"))
+  
+  #turn all into lists of length n objects so plot_prior_posterior() will run evenly
+  traits <- names(trt.match) # not all jagged data was processed
```
~~What does this comment about jagged data mean?~~

Ah, I see now. Because not all of the variables in the priors goes through the meta-analysis because there's no data. Do we still want to plot the prior distributions for those extra variables? I'm not sure if that would be useful at all.

dlebauer replied:
You are correct that they are not as useful when primarily presenting data summaries. But the priors are useful - even in the absence of data - when they show how the model was parameterized.

KristinaRiemer reviewed (approved):
Figures look good!
